Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    it "returns true if svn --version command succeeds" do
+      allow_any_instance_of(Process::Status).to receive(:success?).and_return(true)
```

Comment:
I think it may be worth allowing the `svn` shim to be called. What fails if it is?